QRCoderTests/SvgQRCodeRendererTests.cs

```diff
@@ -103,6 +103,7 @@ public void can_render_svg_qrcode_without_quietzones_hex()
             result.ShouldBe("4ab0417cc6127e347ca1b2322c49ed7d");
         }
 
+#if NETFRAMEWORK || NETSTANDARD2_0 || NET5_0 || NET6_0_WINDOWS
```

Comment:
`#if !NET6_0` would exclude both .NET 6 on Linux and .NET 6 on Windows.  So the ifdef has been fixed to properly run on all applicable test targets